spec/services/idv/profile_maker_spec.rb

```diff
@@ -87,13 +91,59 @@
           fraud_review_needed: false,
           gpo_verification_needed: false,
           deactivation_reason: nil,
+          fraud_provider_status: nil,
         )
 
         expect(profile.active).to eq true
         expect(profile.deactivation_reason).to be_nil
       end
     end
 
+    context 'with fraud pending reason review' do
+      it 'creates an inactive profile with fraud pending reason threatmetrix_review' do
+        profile = subject.save_profile(
+          active: false,
+          fraud_review_needed: false,
```

Comment:
Perhaps this can be combined with the test above, or `fraud_review_needed` should be true here for consistency.